pkg/controller/flinkapplication/controller.go

```diff
@@ -59,15 +59,7 @@ func newReconcilerMetrics(scope promutils.Scope) *reconcilerMetrics {
 }
 
 func (r *ReconcileFlinkApplication) getResource(ctx context.Context, key types.NamespacedName, obj runtime.Object) error {
-	err := r.cache.Get(ctx, key, obj)
```

Comment:
Could you please provide details why `cache` is removed?

Reply:
I suppose that the general reason of operator's inconsistencies is the fact that object specification is mutable during cluster transition from the first stage to end. In order to resolve that I rejected cache usage and introduced infinite loop inside operator. 
In terms of cache I don't have strong opinion. Probably I will get it back when I finish testing my hypothesis.